### ISSUE TITLE
Particles2D initial emission delayed on HTML5 export (WebGL 2)

### ISSUE THREAD

**Issue by jackwilsdon** [godotengine/godot#39654]
**Godot version:** v3.2.1.stable.official (also tried in v3.2.2.rc2.official)

**OS/device including version:** Windows 10 Pro N (Version 1909)

**Issue description:** Emission from a Particles2D node does not start on the first frame, instead waiting for a single lifetime before starting.

**Steps to reproduce:**
1. Open minimal reproduction project and export to HTML5 (or use pre-exported project in `html` folder)
2. Open the exported project in a browser (I tried both Firefox 77.0.1 and Google Chrome 83.0.4103.106)
2. Notice that the emitter only starts emitting particles after 1 second.

When running in either the editor or as an exported Windows Desktop application, the emitter starts immediately.

**Minimal reproduction project:** [emitter-issue.zip](https://github.com/godotengine/godot/files/4801128/emitter-issue.zip)

**Comment by Calinou** [comment]
Judging by the `project.godot` file, this appears to be an issue with the WebGL 2.0 renderer, which we don't support anymore.

**Comment by akien-mga** [comment]
> Judging by the `project.godot` file, this appears to be an issue with the WebGL 2.0 renderer, which we don't support anymore.

We might change our policy on this if we decide to keep GLES3 support in Godot 4.0 as suggested in godotengine/godot-proposals#877, so this might be worth looking into nevertheless.

That being said, it sounds like the typical issue of slow shader compilation for WebGL 2.0 via ANGLE on Windows, for which we don't really have any good solution in mind.

**Comment by Calinou** [comment]
Closing in favor of https://github.com/godotengine/godot/issues/87843.
